debian/control

```diff
@@ -0,0 +1,26 @@
+Source: grpc
+Priority: optional
+Maintainer: Andrew Pollock <apollock@debian.org>
+Build-Depends: debhelper (>= 9), zlib1g-dev, libssl-dev, libprotobuf-dev, protobuf-compiler, libgflags-dev, autoconf, automake, libtool, libgtest-dev, libgoogle-perftools-dev
```

Comment:
> We technically don't depend on autoconf / automake / libtool for the grpc package directly. This is for building protobuf.

Okay I'll take them back out. I would have put them in when I was trying to build extra bits from third_party